addons/base_import_module/models/ir_module.py

```diff
@@ -104,6 +104,10 @@ def _import_module(self, module, path, force=False):
                     if attachment:
                         attachment.write(values)
                     else:
+                        # Do not create a bridge module for this check.
+                        if 'website_id' in IrAttachment._fields:
```

Comment:
Probably need to fix root cause, not consequence.
My guess here is that the overide of write in website irattachment is getting a website from getcurrentwebsite() while it shouldn't -> the context and session sould be clean of any website during the import process, no?

Reply:
I end up writing this:
```py
    def import_module(self):
        # Make sure the import is not website-specific.
        if request:
            request.session['force_website_id'] = None
            request.session.geoip = None
        ...
```
That seems too specific to the current implementation of `get_current_website()`... if yet another rule is added to that method, nobody will think about updating this. Maybe I should revert to resetting the id... or set `not_force_website_id` in the context.

Shouldn't it be clean before any operation that does not involve website ?
E.g. access website 2, add an attachment file to a Contact's chatter... the attachment record is specific to website 2. Sure, the access is not filtering per website so it breaks nothing, but it does not feel correct.

Reply:
Ultimately returned to the basic `values['website_id'] = None` because the alternative was much worse (involving a try/finally and having to setup/clean fields in the session base on the implementation of `get_current_website` - which might evolve)